### PR TITLE
Create post-reference-guide.md

### PR DESCRIPTION
**Pull request by JoeWainer**

### CODE REVIEW

File: student-work/joe-wainer/api-final-project/post-reference-guide.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+| --- | --- | --- | --- | --- |
+| type | The drink for the meal. Either “waterBottle”, “coke”, “lemonade”. | “coke” | string | No |
+| size | Size of the drink. Either “small”, medium”, “large”.  | “medium” | string | No |
+
+## Example
+
+### Request
+
+```json
+curl -H "Content-Type: application/json" -X POST -d'{
```
Comment (Laura-Novich-OBW) [inline student-work/joe-wainer/api-final-project/post-reference-guide.md:39]:
```suggestion
{
```

File: student-work/joe-wainer/api-final-project/post-reference-guide.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            "size":""
+         }
+      },
+      "drink":{
+         "type":"Coke",
+         "size":"large",
+         "ice":"yes"
+      }
+   }
+}'
```
Comment (Laura-Novich-OBW) [inline student-work/joe-wainer/api-final-project/post-reference-guide.md:72]:
```suggestion
}
```

File: student-work/joe-wainer/api-final-project/post-reference-guide.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+### Response
+
+```json
```
Comment (Laura-Novich-OBW) [inline student-work/joe-wainer/api-final-project/post-reference-guide.md:77]:
```suggestion
```http
```

File: student-work/joe-wainer/api-final-project/post-reference-guide.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# POST Reference Guide
+POST is used to send commands to the server. The burger JSON object is included in the POST request which is sent to the kitchen. When an OK response is received, the order is printed in the kitchen so the kitchen staff can start cooking the order. 
+
+# POST/burgerMeal/burger Query Parameters
+
+| Name | Description | Default Value | Data Type | Required |
+| --- | --- | --- | --- | --- |
+| pattyType | The type of patty. Either “beef”, “chicken”, “vegetarian”. | beef | string | Yes |
```
Comment (Laura-Novich-OBW) [inline student-work/joe-wainer/api-final-project/post-reference-guide.md:8]: For all cases use straight quotes for the choices.
For example,
``beef`` vs "beef"

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
changes within

**Review by JoeWainer — COMMENTED** [review]
changed quotation marks to straight

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**